draft-ietf-acme-acme.md

```diff
@@ -472,7 +472,7 @@ can still provide a degree of access control by assigning them
 capability URLs {{?W3C.WD-capability-urls-20140218}}.
 As above, if the server does not allow GET requests for a given
 resource, it MUST return an error with status code 405 "Method Not
-Allowed" and type "malformedRequest".
+Allowed" and type "malformed".
```

Comment:
This appears another time in [line 447](https://github.com/ietf-wg-acme/acme/pull/454/files#diff-8430e2aa241beb4ac49b252db20d4ee8R447).